RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -142,20 +142,51 @@ struct SubscriptionDetailsView: View {
 
     var body: some View {
         VStack(alignment: .leading, spacing: 8) {
-            Text("\(subscriptionInformation.title) - \(subscriptionInformation.durationTitle)")
-                .font(.subheadline)
-                .padding([.horizontal, .top])
-
-            Text("\(subscriptionInformation.price)")
-                .font(.caption)
-                .foregroundColor(Color.gray)
-                .padding(.horizontal)
-
-            if let nextRenewal =  subscriptionInformation.nextRenewalString {
-                Text("\(subscriptionInformation.renewalString): \(String(describing: nextRenewal))")
+            VStack(alignment: .leading) {
+                Text("\(subscriptionInformation.title)")
+                    .font(.headline)
+                Text("\(subscriptionInformation.explanation)")
+                    .frame(maxWidth: 200, alignment: .leading)
                     .font(.caption)
                     .foregroundColor(Color.gray)
-                    .padding([.horizontal, .bottom])
+            }.padding([.bottom], 10)
+
+            HStack(alignment: .center) {
+                Image(systemName: "coloncurrencysign.arrow.circlepath")
```

Comment:
I guess we should do a design review later, but I think these icons look great 🙌

Reply:
We might want to hide these images from accessibility readers with `.accessibilityHidden(true)`. But then again, we might want to address accessibility later.

Reply:
No this is a very good point! I'll do it now otherwise it'll risk being forgotten: https://github.com/RevenueCat/purchases-ios/pull/4072/commits/1ad91a0dacaeb529de1ce5e981ffa96ec5db48af